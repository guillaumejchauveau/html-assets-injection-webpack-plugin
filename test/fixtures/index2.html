<!DOCTYPE html>
<p>a2</p>
