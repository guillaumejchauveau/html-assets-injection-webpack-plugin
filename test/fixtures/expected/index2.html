<!DOCTYPE html>
<link rel="stylesheet" href="./css/app2.css"><p>a2</p>
<script src="./js/app2.js"></script>
